chains/testnet-1/README.md

```diff
@@ -0,0 +1,21 @@
+# testnet-1
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you'll need to provide a signed gentx in a [⚖️ Register Validator issue](https://github.com/okp4/networks/issues). You'll not need to fork the project and make the changes in the genesis yourself, this will be managed by the CI.
```

Comment:
```suggestion
To register your validator node in the `genesis.json` you'll need to provide a signed gentx in a [⚖️ Register Validator issue](https://github.com/okp4/networks/issues). You'll **not** need to fork the project and make the changes in the genesis yourself, this will be managed by the CI.
```
May help those (like me) who read too hastily ^^